fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/CreatePartitionTopNFromWindow.java

```diff
@@ -82,89 +66,24 @@ public Rule build() {
             LogicalWindow<Plan> window = filter.child();
 
             // We have already done such optimization rule, so just ignore it.
-            if (window.child(0) instanceof LogicalPartitionTopN
+            if (filter.getConjuncts().isEmpty()
```

Comment:
if filter.getConjuncts().isEmpty(), how about remove this filter?

Reply:
harmless, can remove it in the future.

Reply:
done